### PR TITLE
Challenges: upload_to attribute added to file fields

### PR DESCRIPTION
**Pull request by deshraj**

### CODE REVIEW

File: apps/challenges/migrations/0006_upload_to_attrbute_added.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    dependencies = [
+        ('challenges', '0005_changed_phase_model'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='challenge',
+            name='evaluation_script',
+            field=models.FileField(default=False, upload_to='evaluation_scripts'),
```
Comment (TroJan) [inline apps/challenges/migrations/0006_upload_to_attrbute_added.py]: @deshraj : forgot to update the migrations. can you please update both folder names here?

Comment (deshraj) [inline apps/challenges/migrations/0006_upload_to_attrbute_added.py]: Can you check the next file? I have updated the migrations.

Comment (TroJan) [inline apps/challenges/migrations/0006_upload_to_attrbute_added.py]: please delete it

**Comment by TroJan** [comment]
@deshraj : looks good. you can merge it.

**Pull request merged**